src/components/FeedbackCard.tsx

```diff
@@ -1,43 +1,36 @@
 import { mdiForum } from "@mdi/js"
 import Icon from "@mdi/react"
 import { memo } from "react"
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+//@ts-ignore
```

Comment:
Consider typing this library in a .d.ts file instead.

Reply:
Will just remove the rat